### PR TITLE
f7x7, f7x9 dfsdm clusters

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

**Comment by burrbull** [comment]
needs https://github.com/stm32-rs/svdtools/issues/88

**Comment by burrbull** [comment]
Rebased & fixed

File: devices/common_patches/dfsdm/f7.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      "DFSDM[0123]_JDATAR":
+        name: JDATAR
+      "DFSDM[0123]_RDATAR":
+        name: RDATAR
+      "DFSDM[0123]_AWHTR":
+        name: AWHTR
+      "DFSDM[0123]_AWLTR":
+        name: AWLTR
+      "DFSDM[0123]_AWSR":
+        name: AWSRR
```
Comment (adamgreig) [inline devices/common_patches/dfsdm/f7.yaml:43]:
```suggestion
        name: AWSR
```

File: devices/common_patches/dfsdm/f7.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+DFSDM:
+  _modify:
+    DFSDM?_RDATAR:
+      alternateRegister: JDATAR
```
Comment (adamgreig) [inline devices/common_patches/dfsdm/f7.yaml:4]: Why `alternateRegister` if they're at different addresses?

**Review by adamgreig — APPROVED** [review]
bors r+

**Pull request merged**